bin/nexus-blobstores-create

```diff
@@ -64,12 +64,12 @@ while [[ $# -gt 0 ]]; do
             -d @-
         status=$?
         if [[ $status -eq 0 ]]; then
-            echo >&2 "INFO Successfully created ${blobstore_type} blobstore: ${blobstore_name}"
+            echo >&2 " OK created"
         else
-            echo >&2 "ERROR Failed to create ${blobstore_type} blobstore: ${blobstore_name}"
+            echo >&2 " FAIL"
```

Comment:
IUF will not be able to recognize this message if ERROR is removed. The details were added for extra information for troubleshooting.